### ISSUE TITLE
Support for PhotoMaker-V2

### ISSUE THREAD

**Issue by Remember2015** [comfyanonymous/ComfyUI#4089]
### Feature Idea

[PhotoMaker](https://github.com/TencentARC/PhotoMaker?tab=readme-ov-file) had version v2 now, according to readme: 
"If the ID fidelity is not enough for you, please try our [PhotoMaker V2](https://huggingface.co/spaces/TencentARC/PhotoMaker-V2) or [stylization application](https://huggingface.co/spaces/TencentARC/PhotoMaker-Style), you may be pleasantly surprised."
I found codes in comfy_extras/nodes_photomaker.py need to be updated, thanks

**Comment by KINGLIFER** [comment]
Here here

**Comment by bigcat88** [comment]
Can we ask at least about the approximate time of adding this to the ComfyUI repository?

To know that if this is not planned in the near future - then we will make workflows based on the [ComfyUI-PhotoMaker-Plus](https://github.com/shiimizu/ComfyUI-PhotoMaker-Plus) node

Thanks.

**Comment by pandayummy** [comment]
Yes, need support for https://huggingface.co/TencentARC/PhotoMaker-V2

photomaker nodes  https://github.com/comfyanonymous/ComfyUI/commit/d1533d9c0f1dde192f738ef1b745b15f49f41e02#diff-c643357dddaaba10bd1db84e4adeff0b2fdcc10be320aa52df892ed67fff0befR8

**Comment by ltdrdata** [comment]
`photomaker-v2` relies on insightface, and including this dependency in ComfyUI is considered very risky, so it is deemed appropriate to keep it as a custom node.

**Comment by JorgeR81** [comment]
The [ComfyUI-PhotoMaker-Plus](https://github.com/shiimizu/ComfyUI-PhotoMaker-Plus) node, mentioned above, also supports, PhotoMaker-V2.

**Comment by pandayummy** [comment]
Just found IPAdapter FaceID easier to setup and use, so will forget about photomaker.
